src/PeopleDashboard.php

```diff
@@ -305,104 +284,35 @@
                     </tr>
-                    <?php foreach ($demographicStats as $demStat) {
-                        $countMale = PersonQuery::create()->filterByFmrId($demStat->getOptionID())->filterByGender(1)->count();
-                        $countFemale = PersonQuery::create()->filterByFmrId($demStat->getOptionID())->filterByGender(2)->count();
-                        $countUnknown = PersonQuery::create()->filterByFmrId($demStat->getOptionID())->filterByGender(0)->count();
-                        $demStatId = $demStat->getOptionID();
-                        $demStatName = $demStat->getOptionName();
-                        $genPop = PersonQuery::create()->count();
-                        if ($countMale != 0) {
-                            ?>
-                            <tr>
-                                <td><a href="v2/people?Gender=1&FamilyRole=<?= $demStatId ?>"><?= $demStatName ?> - <?= gettext('Male') ?></a></td>
-                                <td>
-                                    <div class="progress progress-xs progress-striped active">
-                                        <div class="progress-bar progress-bar-success" style="width: <?= round(($countMale / $genPop) * 100) ?>%" title="<?= round(($countMale / $genPop) * 100) ?>%"></div>
-                                    </div>
-                                </td>
-                                <td><span class="badge bg-green"><?= $countMale ?></span></td>
-                            </tr>
-                            <?php
-                        }
-                        if ($countFemale != 0) {
-                            ?>
-                            <tr>
-                                <td><a href="v2/people?Gender=2&FamilyRole=<?= $demStatId ?>"><?= $demStatName ?> - <?= gettext('Female') ?></a></td>
-                                <td>
-                                    <div class="progress progress-xs progress-striped active">
-                                        <div class="progress-bar progress-bar-success" style="width: <?= round(($countFemale / $genPop) * 100) ?>%" title="<?= round(($countFemale / $genPop) * 100) ?>%"></div>
-                                    </div>
-                                </td>
-                                <td><span class="badge bg-green"><?= $countFemale ?></span></td>
-                            </tr>
-                            <?php
-                        }
-                        if ($countUnknown != 0) {
+                    <?php
+                    foreach(array_keys($familyRoleStats) as $key){
+                        $genderId = $familyRoleStats[$key]['genderId'];;
+                        $roleId = $familyRoleStats[$key]['roleId'];;
+                        $roldGenderName = $key;
+                        $roleGenderCount = $familyRoleStats[$key]['count'];
+
+                        if ($roleGenderCount != 0) {
```

Comment:
can we do strict checking here?

```suggestion
                        if ($roleGenderCount !== 0) {
```

Reply:
Done